Description: Relax ninja version requirement to match version in xenial
Origin: https://chromium.googlesource.com/chromium/src/tools/gn/+/ebe34f81141db077aa69427fd2a7e1c8495a2cf2
Author: Olivier Tilloy <olivier.tilloy@canonical.com>

--- a/tools/gn/tools/gn/ninja_build_writer.cc
+++ b/tools/gn/tools/gn/ninja_build_writer.cc
@@ -278,7 +278,7 @@ bool NinjaBuildWriter::RunAndWriteFile(c
 }
 
 void NinjaBuildWriter::WriteNinjaRules() {
-  out_ << "ninja_required_version = 1.7.2\n\n";
+  out_ << "ninja_required_version = 1.5.1\n\n";
   out_ << "rule gn\n";
   out_ << "  command = " << GetSelfInvocationCommand(build_settings_) << "\n";
   out_ << "  description = Regenerating ninja files\n\n";
